lib/models/bucketentry.js

```diff
@@ -12,10 +12,6 @@ const hmactypes = ['sha512', 'sha256', 'ripemd160'];
 const erasuretypes = ['reedsolomon'];
 
 function isValidIndex(index) {
-  // XXX DEPRECATION IN NEXT MAJOR RELEASE
-  // XXX The determinitic id will be deprecated and the index field will
-  // XXX become required for any new bucket entries. This is to make sure
-  // XXX that each bucket entry will have a unique id.
   if (index) {
```

Comment:
We may want to validate that the index exists, I think this was the plan from before.

Reply:
And this is actually done by calling this `isValidIndex()` function from the `create()` function, right? (Line 117 after changes) Or did you mean something different?

Reply:
I think by removing the if statement for `if (index)`, but will need to check that this is okay.

Reply:
The `PATCH` method in the bridge will check if there is an index. If there isn't one, it will return Unprocessable Entity Error. So it would be safe to continue creating bucket entries without index. Simply, it won't be possible to rename them afterwards.

However, if you want to force clients to create only bucket entries with index, we should remove the `if (index)`.